base/constants.go

```diff
@@ -57,17 +57,26 @@ const DelayBetweenSourceAndTarget uint64 = 2
 const CheckpointInterval = 600
 
 // length of mutation metadata + body, which consists of
-//  seqno   - 8 bytes
-//  revId   - 8 bytes
-//  cas     - 8 bytes
-//  flags   - 4 bytes
-//  expiry  - 4 bytes
-//  opCode  - 2 bytes
-//  hash    - 64 bytes
-const BodyLength = 98
+//  seqno    - 8 bytes
+//  revId    - 8 bytes
+//  cas      - 8 bytes
+//  flags    - 4 bytes
+//  expiry   - 4 bytes
+//  opCode   - 2 bytes
+//  datatype - 2 byte
```

Comment:
is it better to use 1 byte, since it always interpreted as a uint8 in code

Reply:
Reason I used 2 bytes is because binary.BigEndian only has PutUint16, but no PutUint8